lib/ansible/modules/windows/win_acl.ps1

```diff
@@ -175,7 +213,7 @@ Try {
             Set-Attr $result "changed" $true;
         }
         Catch {
-            Fail-Json $result "an exception occurred when adding the specified rule"
+            Fail-Json $result "an exception occurred when adding the specified rule - $_.Exception.Message"
```

Comment:
+1 for adding these, should have been there to begin with for more details error info

You will need to change to `$($_.Exception.Message)` for that to expand correctly.  Or else you'll get 
```
an exception occurred when adding the specified rule - .Exception.Message
```
every time